src/components/styles/sprite-canvas.css.ts

```diff
@@ -0,0 +1,11 @@
+import { style } from '@vanilla-extract/css';
+
+export const spriteCanvasStyle = style({
+  height: `auto`,
+//   background: #aaaaaa,
```

Comment:
I left this comment here as a reminder that we should figure out how to actually do this correctly at some point...